tests/Unit/Invariants/EmailTriggerInvariantTest.php

```
@@ -0,0 +1,201 @@
+				$trimmed = ltrim( $line );
+				if ( '' === $trimmed || str_starts_with( $trimmed, '//' ) || str_starts_with( $trimmed, '*' ) || str_starts_with( $trimmed, '/*' ) ) {
+					continue;
+				}
+				if ( preg_match( $pattern, $line, $m ) ) {
+					$rel   = ltrim( str_replace( $this->plugin_root(), '', $file ), '/' );
+					$out[] = array( $m[1], $m[2], $rel . ':' . ( $lineno + 1 ) );
+				}
+			}
+		}
+		return $out;
+	}
```

PHPMD (2.15.0)

[error] The method extract_call_sites is not named in camelCase. (undefined)

(CamelCaseMethodName)